releasenotes/notes/replace-requirment-files-and-setup.py-with-pyproject.toml-4db05996dd99827c.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
We don't do releases of this template repo, so I don't think we need a release note here.

Reply:
No problem, let me remove that now.